Sources/SearchableCollectionViewController/SearchBarView.swift

```diff
@@ -1,34 +1,36 @@
 import Foundation
 import UIKit
 
-class SearchBarView: UICollectionReusableView {
-    
-    static let height: CGFloat = 44.0
-    static let kind = String(describing:SearchBarView.self)
-    static let defaultSearchBackgroundColor = UIColor(hex: "00C365")
-    
-    dynamic open var searchBackgroundColor: UIColor? = SearchBarView.defaultSearchBackgroundColor {
-        didSet { searchBar.backgroundColor = searchBackgroundColor }
+open class SearchBarView: UICollectionReusableView {
```

Comment:
I did answer. Multiple times. But here I go again.

<img width="429" alt="screen shot 2017-04-24 at 16 16 30" src="https://cloud.githubusercontent.com/assets/54423/25341537/6bfa52e0-2909-11e7-96fd-5dafb28d458d.png">

Right now search doesn't even _work_, because there's no way to set the search bar delegate, so we can act on changes and update the content. 👍

It also doesn't integrate a search controller, but not sure if that is even technically possible. 🤔 Something for the future?